util/brew-update

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env ys-0
 
-releases-url =: 'https://github.com/yaml/yamlscript/releases'
-brew-url =: 'git@github.com:yaml/homebrew-yamlscript'
+releases-url =: "https://github.com/yaml/yamlscript/releases"
+repo =: ENV.GITHUB_REPOSITORY_OWNER || 'yaml'
+brew-url =: "https://github.com/$repo/homebrew-yamlscript"
 
 defn main(version):
   check-version: version
   download-releases: version
   update-brew: version
 
+defn sha256sum(file):
+  os =: System/getProperty("os.name")
+  cond:
```

Comment:
`cond` is better as `if` when only 1 condition.
I'd refactor this to something like:
```
os =: System/getProperty("os.name")

defn sha256sum(file):
  sh:
    if os =~ /Mac\s?OS/:
      then: "shasum -a 256 $file"
      else: "sha256sum $file"
```

Reply:
Note: `os` is at the file/namespace scope and can be used elsewhere.

Reply:
> `cond` is better as `if` when only 1 condition.

Yeah I started out with an `if`, but figured in future it might need another case for Windows… So went with `cond`.